include/exadg/structure/user_interface/application_base.h

```diff
@@ -70,12 +70,14 @@ class ApplicationBase
   {
     parse_parameters();
 
-    // parameters
-    set_parameters();
 #ifdef DEAL_II_WITH_TRILINOS
-    // Overwrite default MLOperatorType::Laplace to include rotational rigid body modes.
+    // Overwrite default MLOperatorType::Laplace to include rotational rigid body modes;
```

Comment:
sets default for structure applications to `Elasticity`, but later the user can overwrite

Reply:
In my opinion this is the same style as before ("implicitly" defined default value with adjustments in only some modules; is e.g. the FSI module correct now?). The above comment "overwrite default MLOperatorType::Laplace" is not safe, one can easily change the default-value in ExaDG (which happens at a completely different place and affects all the modules) and this comment will be falsified (and also the code no longer works as expected). I also think we should not start with such problem specific adjustments and ifdefs related to Trilinos on such a high level (application class). Note that the structure solver is used in other modules such as FSI: can a user understand (for the present code) what will happen there? Hence, I think we need to set this parameter at a more central place in `Structure::Operator::setup_preconditioner()` for example, see https://github.com/exadg/exadg/blob/711cb015f0a56047dcd4c2f91a9b3d349dbcdd6e/include/exadg/structure/spatial_discretization/operator.cpp#L323 (To be on the safe side, the respective preconditioner implementations could add asserts such that the parameter value received by the preconditioner class agrees with the expected value for a certain PDE model problem.) Then, there is no need to print this parameter to the screen output, because it is an internal parameter.

Reply:
I will remove it and not print it. and set it in the suggested place.